scratch.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
I didn't mean to include this file, but I guess it's good to have for reference. Let's rename it and move it into the notebooks folder. "tech_stack_vis_testing.ipynb"